devices/wholeBodyDynamics/KnownExternalWrench.cpp

```diff
@@ -0,0 +1,127 @@
+#include "KnownExternalWrench.h"
+
+namespace wholeBodyDynamics
+{
+
+KnownExternalWrench::KnownExternalWrench::KnownExternalWrench()
+{
+}
+
+KnownExternalWrench::KnownExternalWrench(std::string parsedframeName, std::string parsedType, std::vector<double> parsedDirection, std::vector<double> parsedPosition)
+{
+    frameName = parsedframeName;
+    type = parsedType;
+    position = parsedPosition;
+    direction = parsedDirection;
+    
+    if(type=="full")
+    {
+        noOfVariables = 6;
```

Comment:
Here lies the confusion, whether it is known variables or unknown variables? and variables in which component? Better to be descriptive than be confused.

Reply:
Correct.
related comment https://github.com/robotology/whole-body-estimators/pull/48#discussion_r393002080